README.md

```diff
@@ -53,10 +53,5 @@ hdmi_drive=1
 ```
 (480x320 seems to end up wrong if set as cvt)
 
-Create venv and install requirements:
-```sh
-python -m venv --system-site-packages venv
-pip install pyotp qrcode
-```
-Copy `.xsession` over.
-
+## Software
```

Comment:
As a pure python softworker, it's really uncommon for me to see a `software` folder. I typically know to see the python project at top-level, then a `src` folder for source code. Having all that in a subfolder feels strange 😅